src/Microsoft.DotNet.TestFramework/Microsoft.DotNet.TestFramework.TestInstance.cs

```diff
@@ -11,6 +11,9 @@ namespace Microsoft.DotNet.TestFramework
 {
     public class TestInstance
     {
+        // made tolower because the rest of the class works with normalized tolower strings
+        private static readonly IEnumerable<string> BuildArtifactBlackList = new List<string>() {".IncrementalCache", ".SDKVersion"}.Select(s => s.ToLower());
```

Comment:
I also still want to call out that I don't think calling ToLower on file paths is correct cross-platform, since some platforms have case sensitive file systems.